plugins/async-node/core/src/index.ts

```diff
@@ -70,6 +77,21 @@ export class AsyncNodePluginPlugin implements AsyncNodeViewPlugin {
 
   private currentView: ViewInstance | undefined;
 
+  private updateAsyncFunc(
```

Comment:
Nit: Naming for this - I wouldn't usually expect a member function to have `Func` in the name since it's redundant. `handleAsyncUpdate` might fit better, but open to other ideas!

Reply:
done